### DISCUSSION TITLE
Pediatrics scaling

### DISCUSSION THREAD

**Discussion by HaiXxxxxx** [Open-Systems-Pharmacology/Forum#1768] [Questions & Problems]
Hello, everyone. When extrapolated to pediatrics, do we must define clearances from specific metabolic enzymes instead of unspecific hepatic clearance?

**Comment by tobiasK2001** [comment]
Hi,
yes that way you can take advantage from the ontogeny functions to describe  enzyme maturation.

Best, Tobias

**Reply by HaiXxxxxx** [reply]
> Hi, yes that way you can take advantage from the ontogeny functions to describe enzyme maturation.
> 
> Best, Tobias
Thank you!
